src/main/haskell/kore/test/Test/Kore/Step/Function/UserDefined.hs

```diff
@@ -92,6 +92,28 @@ test_userDefinedFunction =
                 (asApplication (metaF (x PatternSort)))
             )
         )
+    , testCase "Cannot apply step with unsat axiom pre-condition"
```

Comment:
@virgil-serbanuta Is this what you had in mind for an axiom condition test in `UserDefined`?